core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala

```diff
@@ -246,8 +246,10 @@ class SimpleAclAuthorizer extends Authorizer with Logging {
   }
 
   private def logAuditMessage(principal: KafkaPrincipal, authorized: Boolean, operation: Operation, resource: Resource, host: String) {
-    val permissionType = if (authorized) "Allowed" else "Denied"
-    authorizerLogger.debug(s"Principal = $principal is $permissionType Operation = $operation from host = $host on resource = $resource")
+    if (authorized)
+      authorizerLogger.debug(s"Principal = $principal is Allowed Operation = $operation from host = $host on resource = $resource")
+    else
+      authorizerLogger.warn(s"Principal = $principal is Denied Operation = $operation from host = $host on resource = $resource")
```

Comment:
We can probably extract the message instead of duplicating it. Also, should we use `warn` or `info` for denials?

Reply:
That seems OK. I don't like `WARN` because that implies that something is not quite right, but denials don't necessarily mean that. cc @rajinisivaram for a second opinion.

Reply:
Yes, `INFO` sounds right for this case.

Reply:
Thanks for the review. Updated the PR.

Reply:
@ijuma  @rajinisivaram pinging for review